common/install_cpython.sh

```diff
@@ -40,7 +40,11 @@ function do_cpython_build {
     mkdir -p ${prefix}/lib
 
     # -Wformat added for https://bugs.python.org/issue17547 on Python 2.6
-    CFLAGS="-Wformat" ./configure --prefix=${prefix} --disable-shared $unicode_flags > /dev/null
+    if [[ -z  ${WITH_OPENSSL} ]]; then
+        CFLAGS="-Wformat" ./configure --prefix=${prefix} --disable-shared $unicode_flags > /dev/null
```

Comment:
btw if you have issues with pip missing for some versions, you can add `--with-ensurepip=install` to this